frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -194,6 +211,100 @@ const updateItemAppConnectTo = (
   return k8sPatch(model, item, patch);
 };
 
+export const createServiceBinding = (
+  source: K8sResourceKind,
+  target: K8sResourceKind,
+): Promise<any> => {
+  if (!source || !target || source === target) {
+    return Promise.reject();
+  }
+
+  const targetName = _.get(target, 'metadata.name');
+  const sourceName = _.get(source, 'metadata.name');
+  const namespace = _.get(source, 'metadata.namespace');
+  const targetResourceGroup = _.split(_.get(target, 'metadata.ownerReferences[0].apiVersion'), '/');
+  const targetResourceKind = _.get(target, 'metadata.ownerReferences[0].kind');
+  const targetResourceRefName = _.get(target, 'metadata.ownerReferences[0].name');
+  const applicationSelectorLabels = getApplicationSelectorLabels(targetName);
+
+  const serviceBindingRequest = {
+    apiVersion: 'apps.openshift.io/v1alpha1',
+    kind: 'ServiceBindingRequest',
+    metadata: {
+      name: `${sourceName}-${targetName}-sbr`,
+      namespace,
+    },
+    spec: {
+      applicationSelector: {
+        matchLabels: { ...applicationSelectorLabels },
+        group: 'apps.openshift.io',
+        version: 'v1',
+        resource: 'deploymentconfigs',
+      },
+      backingServiceSelector: {
+        group: targetResourceGroup[0],
+        version: 'v1alpha1',
```

Comment:
This cannot be hard coded.

Reply:
Yes, this information is available in the csv - example, https://github.com/operator-framework/community-operators/blob/master/community-operators/apicurito/0.1.0/apicuritooperator.v0.1.0.clusterserviceversion.yaml#L27